__mocks__/client-s3.ts

```diff
@@ -1,5 +1,23 @@
+export let s3ObjectsPut = {};
```

Comment:
This is used to verify that the keys of the objects uploaded to s3 have the auto versioned virtual folder in them